android/cpp/datasets/ade20k.cc

```diff
@@ -179,7 +174,8 @@ std::vector<uint8_t> ADE20K::ProcessOutput(const int sample_idx,
           if (p == g) {
             true_positive++;
           } else if (p == c) {
-            if ((g > 0) && (g <= num_classes_)) false_positive++;
+            if ((g > 0) && (g <= num_classes_))
```

Comment:
Did the clang-format version change or were these not previously formatted? It would be better that all if statements used parenthesis.

Reply:
Looks like a Jenkins job that should check files formatting did it incorrectly. The job has been fixed